lms/templates/fields/field_dropdown.underscore

```diff
@@ -8,9 +8,9 @@
     <i class="u-field-icon icon fa <%- iconName %> fa-fw" area-hidden="true" ></i>
 <% } %>
 
-<span class="u-field-value">
+<span class="u-field-value" tabindex="0" role="listbox" aria-expanded="false">
```

Comment:
This is now breaking all of the listboxes on the "Account Settings" page.  Those are all regular select elements and the role override here is conflicting with them.  It's also not working very well on the "My Profile" page either so we may need to take a different approach.  LEt's roll this back for now.